src/sentry/issues/occurrence_consumer.py

```diff
@@ -264,8 +265,14 @@ def _process_message(
             txn.set_tag("project_id", project.id)
             txn.set_tag("project_slug", project.slug)
 
-            if not features.has("organizations:profile-blocked-main-thread-ingest", organization):
-                metrics.incr("occurrence_ingest.dropped_feature_disabled", sample_rate=1.0)
+            if occurrence_data["type"] not in PROFILE_FILE_IO_ISSUE_TYPES or not features.has(
+                "organizations:profile-blocked-main-thread-ingest", organization
```

Comment:
Is `profile-blocked-main-thread-ingest` flag being used for `ProfileFileIOGroupType` as well?